contrib/kcp.code-workspace

```diff
@@ -121,6 +121,29 @@
 				"env": { "GOMOD": "${workspaceFolder:kcp}/go.mod", },
 				"cwd": "${workspaceFolder:kcp}",
 				"args": ["-kubeconfig", "${workspaceFolder:kcp}/.kcp/admin.kubeconfig", "-context", "admin"],
+			},
+			{
```

Comment:
nit: can we prefer 

a) scripts
b) end-to-end tests

for this sort of thing, and have VSCode run the common script(s) so that we can all share these?

Reply:
That's useful for debugging. But I might just remove this in the final PR.

And yes, I assume that having a more general way to share info between VSCode debugging launch configs and CLI commands might be interesting (for another PR ?)